locales/en/moves.json

```diff
@@ -151,12 +151,16 @@
     }
   },
   "confirmation": {
-    "page_title": "Move scheduled for {{name}}",
+    "page_title_requested": "Move scheduled for {{name}}",
```

Comment:
I would probably leave this line as is and tweak when the context is sent, that way the default remains the `scheduled` for now and we don't have to add a line here for every status.

Reply:
In fact, after checking, if it doesn't find the context, it will use the fallback.

So if we leave this line unchanged, if it doesn't find the status it will use that translation. Then we can define a context line for any status where we want to override the default message.